### ISSUE TITLE
Console error/script error related to BroadcastChannel JS

### ISSUE THREAD

**Issue by adrienne** [nystudio107/craft-seomatic#1055]
### Describe the bug

When saving a new entry for the first time, the console shows the following error:
`Uncaught SyntaxError: Unexpected token ','`

Clicking through in the console to the full context, we see that the error is happening here:

```js
if (typeof BroadcastChannel !== 'undefined') {
    (new BroadcastChannel('DraftEditor')).postMessage({
        event: 'saveDraft',
        canonicalId: 9798503,
        draftId: ,     // <-- this is the line marked in devtools as having the error
        isProvisionalDraft: false,
    });
};
```

It looks to me like it can't find the draftId, and I don't know what's causing that problem, but it seems like there should be a way to handle this error so it doesn't break? 

This doesn't seem to cause any problems with SEOMatic specifically, but the error causes a bunch of other scripts to fail. 

I don't *know* whether this is SEOMatic code, but that's the only place i can find reference to `BroadcastChannel`. 

Resaving the entry again, even with no changes, resolves the error.

### To reproduce
Steps to reproduce the behaviour:
1. Create a new entry
2. Save the entry
3. See the error in the console

### Expected behaviour

It shouldn't throw an error if it can't get a draft ID.

### Versions
- Plugin version: 3.4.23
- Craft version: Craft Pro 3.7.30.1

**Comment by adrienne** [comment]
False alarm, Brandon said it's on their end.

**Comment by khalwat** [comment]
Specifically, it was this: https://github.com/craftcms/cms/commit/554542e5450ccd62bc89783945f3637667bd9313

Fixed in Craft CMS 3.7.31
